model/src/w3iorsmd.F90

```diff
@@ -741,7 +747,13 @@ SUBROUTINE W3IORS ( INXOUT, NDSR, DUMFPI, IMOD, FLRSTRT )
       !
       ! Reading spectra
       !
-      IF ( TYPE.EQ.'WIND' .OR. TYPE.EQ.'CALM' ) THEN
+      IF ( TYPE.EQ.'FTCH') THEN
+        RPOS  = 1_8 + LRECL*(3-1_8)
+        READ(NDSR, POS=RPOS,ERR=802,IOSTAT=IERR) FETCH
```

Comment:
After a recent PR we need to update this to be: 

            READ(NDSR, POS=RPOS,IOSTAT=IERR) FETCH
            IF (IERR.NE.0) CALL EXTIOF(NDSE,IERR,'W3IORS','',30)